x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts

```diff
@@ -147,6 +145,12 @@ export class EnterpriseSearchRequestHandler {
     };
   }
 
+  getBodyAsString(body: object | Buffer): string | undefined {
```

Comment:
If I can suggest a comment block for this - this is a pretty hefty file and has a lot of moving pieces, so I've erred on the side of explaining more business logic where possible. What do you think about:


```suggestion
  /**
   * There are a number of different expected incoming bodies that we handle & pass on to Enterprise Search for ingestion:
   * - Standard object data (should be JSON stringified)
   * - Empty (should be passed as undefined and not as an empty obj)
   * - Raw buffers (passed on as-is, occurs when using the `skipBodyValidation` lib helper)
   */
  getBodyAsString(body: object | Buffer): string | undefined {
```

Reply:
Just wanted to follow up on this one - what were your thoughts on a small unit test for this helper?

Reply:
It's covered currently, yeah? Can we just make it a private field and rely on the main tests?

Reply:
Sorry, i thought I had merged these changes in, guess I didn't.

Reply:
Sorry, i thought I had merged these changes in, guess I didn't.

Reply:
OK, no worries!